1.0/docs/devguide/properties.md

```diff
@@ -123,7 +123,7 @@ for more information.
 
 The value is interpreted as a method name to be invoked when the property value 
 changes. Note that unlike in 0.5, <strong>property change handlers must be registered 
-explicitly.</strong> The <code><var>propertyName</var>-changed</code> method will not be 
+explicitly.</strong> The <code><var>propertyName</var>Changed</code> method will not be 
```

Comment:
I read this as referring to the 0.5 style implicit on change functions rather than any kind of explicitly named function